app/views/geocoder/search.html.erb

```diff
@@ -3,10 +3,17 @@
 <%= render "sidebar_header", :title => t("site.sidebar.search_results") %>
 
 <% @sources.each do |source| %>
-  <h4>
-    <%= t(".title.results_from_html", :results_link => link_to(t(".title.#{source}"),
-                                                               t(".title.#{source}_url"))) %>
-  </h4>
+  <% if source.include?("osm_nominatim") %>
```

Comment:
Rather than having this look at the name of the source it would be better to make `@sources` an array of hashes, each with a `:name` and an optional `:url` key.

Reply:
Dear @tomhughes ,

Thank you very much for your review and comments. I've just made above updates. Please, let me know if you have idea how we could improve it further.

Thanks,
Nenad.